api/v1/storagecluster_types.go

```diff
@@ -686,3 +690,14 @@ func (r *StorageCluster) NewToolsDeployment(tolerations []corev1.Toleration) *ap
 		},
 	}
 }
+
+// OptimizeDisasterRecovery returns true if any RDR optimizations are required
+func (r *StorageCluster) OptimizeDisasterRecovery() bool {
+	if annotation, found := r.GetAnnotations()[DisasterRecoveryOptimizations]; found {
+		if annotation == "true" {
+			return true
+		}
+	}
+
+	return false
+}
```

Comment:
Let's move this to `controllers/storagecluster/cephcluster.go` or to the utility package `controllers/util/`

Reply:
moved it to `controllers/storagecluster/cephcluster.go`